### ISSUE TITLE
Auto delete KFrags for expired policies

### ISSUE THREAD

**Issue by tuxxy** [nucypher/nucypher#186]
When policies expire, the KFrag for a given policy needs to be deleted.

How should we set this up? We need a scheduling system for this. Is there something more simple we can use?

**Comment by KPrasch** [comment]
A good entry point for this might be a conditional right before Ursula carries out a workout

**Comment by michwill** [comment]
To clarify, currently we get a key regardless of expiration:
```
policy_arrangement = session.query(PolicyArrangement).filter_by(id=arrangement_id).first()
```
Instead, we should be doing something like:
```
policy_arrangement = session.query(PolicyArrangement).filter_by(id=arrangement_id, PolicyArrangement.expiration >= datetime.utcnow()).first()
```

As a next step, we should periodically clean up old expirations (having an index on expiration)

**Comment by tuxxy** [comment]
Even better, we have a task queue which automatically expires KFrags.

**Comment by KPrasch** [comment]
@tuxxy - Is this still open?

**Comment by KPrasch** [comment]
Closed in #1747
